test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -755,23 +747,8 @@ public void EntryPointPipelineEnsemble()
                     new RandomFourierFeaturizingTransformer.ColumnInfo("Features", "Features2", 10, false),
                 }).Fit(data).Transform(data);
 
-                data = ColumnConcatenatingTransformer.Create(Env, new ColumnConcatenatingTransformer.Arguments()
-                {
-                    Column = new[] { new ColumnConcatenatingTransformer.Column() { Name = "Features", Source = new[] { "Features1", "Features2" } } }
-                }, data);
-
-                data = ValueToKeyMappingTransformer.Create(Env, new ValueToKeyMappingTransformer.Arguments()
-                {
-                    Column = new[]
-                    {
-                        new ValueToKeyMappingTransformer.Column()
-                        {
-                            Name = "Label",
-                            Source = "Label",
-                            Sort = ValueToKeyMappingTransformer.SortOrder.Value
-                        }
-                    }
-                }, data);
+                data = new ColumnConcatenatingTransformer(Env, "Features", new[] { "Features1", "Features2" }).Transform(data);
```

Comment:
>data = new ColumnConcatenatingTransformer(Env, "Features", new[] { "Features1", "Features2" }).Transform(data); [](start = 16, length = 111)

me looking at this and thinking that we should make the Transformer constructors internal.

Is there any benefit to leaving them public? (i know we've had the conversation for the Predictors/ModelParams and that made sense.)

If it is more than a yes/no answer, i can create an issue and we can discuss it there.

Reply:
I think idea was if it's a trivial estimator, it's ok to have public constructor for it's transformer.
Estimator is become pointless for them unless you try to build chain of estimators.

---
In reply to: [246171719](https://github.com/dotnet/machinelearning/pull/2031#discussion_r246171719) [](ancestors = 246171719)

Reply:
ok, thanks.

---
In reply to: [246172551](https://github.com/dotnet/machinelearning/pull/2031#discussion_r246172551) [](ancestors = 246172551,246171719)